docs/README.md

```diff
@@ -20,7 +20,7 @@ You will create/deploy:
 * A GitHub App
 * A number of secrets
 * A stack of OpenFaaS functions via stack.yml
-* Customise limits for Swarm or K8s
```

Comment:
The British spelling is valid.

Reply:
I hope you are not offended it just gave me green underlining ..